Run7/step02/C_protocol.cfg

```diff
@@ -0,0 +1,106 @@
+[ACQUIRE]
+bias1
+bias2
+bias3
+bias4
+dark
+sflat
+lambda
+flat
+persistence
+[BIAS1]
+ACQTYPE=bias
+ANNOTATION=0 sec extra delay
+COUNT= 5
+EXTRADELAY = 0
+
+[BIAS2]
+ACQTYPE=bias
+ANNOTATION=15 sec extra delay
+COUNT= 5
+EXTRADELAY = 15.0
+
+[BIAS3]
+ACQTYPE=bias
+ANNOTATION=30 sec extra delay
+COUNT= 5
+EXTRADELAY = 30
+
+[BIAS4]
+ACQTYPE=bias
+ANNOTATION=60 sec extra delay
+COUNT= 5
+EXTRADELAY = 60
+
+[DARK]
+SHUTTER = OPEN  # Shutter opens at start of acqusition, closes at end
+BCOUNT=    1    # number of bias frames per dark set
+DARK=30  2,       # integration time and image count for dark set
+     180 2,
+     360 2
+
+[SFLAT]
+SHUTTER = OPEN  # Shutter opens at start of acqusition, closes at end
+FILTERCONFIG = /home/ccs/bot-eotest-configs/Run7/calib.cfg
+BCOUNT=   0    # number of bias frames per superflat set
+HILIM =15.0   # maximum seconds for a flat field exposure
+LOLIM =0       # minimum seconds for a flat field exposure
+#SFLAT=nm750  1000     25  ND_OD1.0,   # wavelength filter, signal(e-), count, ND filter#
+#      nm750  50000    10  empty    # wavelength filter, signal(e-), count, ND filter#
+SFLAT=nm750  1000     5,   # wavelength filter, signal(e-), count
+      nm750  50000    5    # wavelength filter, signal(e-), count
+
+[LAMBDA]
+SHUTTER = OPEN  # Shutter opens at start of acqusition, closes at end
+IMCOUNT=   1    # number of images at each wavelength
+BCOUNT=    0    # number of bias frames per wavelength image
+HILIM =15.0    # maximum seconds for a flat field exposure
+LOLIM =0        # minimum seconds for a flat field exposure
+FILTERCONFIG = /home/ccs/bot-eotest-configs/Run7/calib.cfg
+
+LAMBDA = blue   10000,  # wavelength filter, expected e/pixe
+         nm750  10000,
+         red    10000,
+         uv     10000,
+         nm960  10000,
+         nm850  10000
+
+[FLAT]
+SHUTTER = OPEN  # Shutter opens at start of acqusition, closes at end
+BCOUNT=1        # number of bias frames per flat image
+WL    =nm750    # wavelength filter to use for the flats
+HILIM =15.0    # maximum seconds for a flat field exposure
+LOLIM =0        # minimum seconds for a flat field exposure
+FILTERCONFIG = /home/ccs/bot-eotest-configs/Run7/calib.cfg
+
+# DARKINTERRUPT = True # If True, will take darks between flat images
+# DARKINTERRUPTLIST = 30  5 # integration time and image count for dark set, formatted the same as the DARK acquisition
+
+#
+# Below has pairs from 100-200,000 e/pixel, log spaced with
+# four overlapping pairs around the transition between
+# ND_OD1.0 and empty (568-1090 e/pixel)
```

Comment:
Please update the comment to remove the reference to overlapping pairs.  (I don't see any implemented below.)